sql/ddl_internal.sql

```diff
@@ -565,3 +565,11 @@ BEGIN
     END LOOP;
 END
 $BODY$;
+
+
+--documentation of these function located in chunk_index.h
+CREATE OR REPLACE FUNCTION _timescaledb_internal.chunk_index_recreate_create(chunk_index_oid OID) RETURNS OID
```

Comment:
Weird name, why not simply `chunk_index_clone()` or something?

Reply:
done